cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -66,38 +68,26 @@ public String getCronEntry() {
   }
 
   @Override
-  public boolean equals(Object o) {
-    if (this == o) {
+  public int hashCode() {
+    return Objects.hashCode(name, description, cronEntry);
```

Comment:
Changed to use internal (in `cdap-api`) `Objects`. This change was needed b/c in the list adapters endpoint, some of the `Schedule` fields can be empty.